src/main/java/redis/embedded/RedisExecProvider.java

```diff
@@ -2,18 +2,22 @@
 
 import com.google.common.base.Preconditions;
 import com.google.common.collect.Maps;
-import redis.embedded.util.Architecture;
-import redis.embedded.util.JarUtil;
-import redis.embedded.util.OS;
-import redis.embedded.util.OsArchitecture;
+import redis.embedded.util.*;
 
 import java.io.File;
 import java.io.IOException;
 import java.util.Map;
+import java.util.regex.Pattern;
 
 public class RedisExecProvider {
 
-    private final Map<OsArchitecture, String> executables = Maps.newHashMap();
+    private final Map<OsArchitecture, RedisExecutor> executables = Maps.newHashMap();
+    public final static Pattern REDIS_READY_PATTERN = Pattern.compile(
```

Comment:
More of nitpick than anything, but could you name it `DEFAULT_REDIS_READY_PATTERN` ?
That way we really convey that it's overridable.